content/influxdb/v2.0/influxdb-templates/cloud.md

```diff
@@ -15,9 +14,48 @@ influxdb/v2.0/tags: [templates]
 products: [cloud]
 ---
 
-Use the [`influx` command line interface (CLI)](/influxdb/v2.0/reference/cli/influx/)
-to apply and manage templates in your InfluxDB Cloud account.
+To use templates in InfluxDB Cloud, you have a couple options:
+
+- [Install and customize templates directly in the Cloud UI](#install-and-customize-a-template-in-the-cloud-ui)
```

Comment:
we use the term Apply in the cli and Install in the UI. I think it's out of scope for this PR, but we should probably unify the terms to one or the other - it's a bit confusing that there are two words that mean the same thing.

Reply:
I'll update globally once we update the corresponding terms in the UI